Documentation/subcommands/stop.md

```diff
@@ -0,0 +1,3 @@
+# rkt stop
+
+Given a list of pod UUIDs, rkt stop will shut them down cleanly.
```

Comment:
It should be explained what "shut them down cleanly" means, and compare with `--force`.

The app in the pod receives a TERM signal and after a timeout a KILL signal.

Is the post-stop event handler executed?